applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -290,7 +286,7 @@ static int lc3_config_cb(struct bt_conn *conn, const struct bt_audio_ep *ep, enu
 			else if (dir == BT_AUDIO_DIR_SOURCE) {
 				LOG_DBG("BT_AUDIO_DIR_SOURCE");
 				print_codec(codec);
-				sources[configured_source_stream_count++].stream = audio_stream;
+				sources[0].stream = audio_stream;
```

Comment:
What happens if there are two bidir?

Reply:
Since this is on headset side, so there won't be two bidir scenario (at this moment)